### PR TITLE
Create flatpak app openerp-client (gtk2 desktop application)

### PR DESCRIPTION
**Pull request by rga-odoo**

### CODE REVIEW

**Comment by bilelmoussaoui** [comment]
You're not supposed to include in-tree the shared-modules, you should use git submodules instead

File: org.flatpak.openerp.json [hunk 1]
```diff
@@ -0,0 +1,169 @@
+{
+  "app-id": "org.flatpak.openerp",
```
Comment (bilelmoussaoui) [inline org.flatpak.openerp.json:2]: Doesn't sound like the right id.

Comment (LeandroStanger) [reply org.flatpak.openerp.json:2]:
```suggestion
  "app-id": "com.odoo.Openerp",
```

**Comment by rga-odoo** [comment]
@bilelmoussaoui I have added shared modules as git submodules

File: org.flatpak.openerp.json [hunk 2]
```diff
@@ -0,0 +1,169 @@
+{
+  "app-id": "com.odoo.Openerp",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "19.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "openerp-client",
+  "finish-args": [
+    "--socket=x11",
```
Comment (nedrichards) [inline org.flatpak.openerp.json:8]: you need to add --share=ipc if you use x11

**Comment by nedrichards** [comment]
bot, build com.odoo.Openerp

File: org.flatpak.openerp.json [hunk 3]
```diff
@@ -0,0 +1,169 @@
+{
```
Comment (barthalion) [inline org.flatpak.openerp.json:1]: The file should be renamed to match app-id.

**Comment by Eonfge** [comment]
Hey, I've upstreamed a shared module for python, which contains pygtk. Youre project would be an ideal candiate to use pygtk from the /shared-modules/ since it saves us all from double maintenance:

https://github.com/flathub/shared-modules/tree/master/pygtk

**Pull request closed without merging**